README.md

```diff
@@ -7,7 +7,8 @@ This repository allows you to install [Apache Airflow](https://airflow.apache.or
 - MySQL or PostgreSQL database in which to store Airflow metadata.
 
 ### Currently Supported Versions of Airflow
-- Airflow 1.10
+- Airflow 1.9.0
```

Comment:
Why did this get added?  Have we actually built the Airflow 1.9.0 parcels using the new Docker method?